app/models/model_cache.rb

```diff
@@ -17,15 +17,15 @@ def comments
   end
 
   def discussion_for(discussion_id)
-    discussions.find_by id: discussion_id
```

Comment:
This was N+1'ing because `find_by` runs a DB query, but we already have the records we want stored in memory